Update more tests to conform to new SameSite cookie behavior

charliewolfe opened the pull request:
#### 88e0f3aad1e007427cb75be6c5808cceda4fa384
<pre>
Update more tests to conform to new SameSite cookie behavior
<a href="https://rdar.apple.com/125572233">rdar://125572233</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=276459">https://bugs.webkit.org/show_bug.cgi?id=276459</a>

Reviewed by Matthew Finkel and Sihui Liu.

This addresses most tests that are failing due to the SameSite lax by default change in the macOS Sequoia
and iOS 18 betas. Some tests still fail and will need to be rebased when the bots are updated. Others may
require test runner changes or further test redesigns.

* LayoutTests/http/tests/contentextensions/block-cookies-send.https-expected.txt: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-send-expected.txt.
* LayoutTests/http/tests/contentextensions/block-cookies-send.https.html: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-send.html.
* LayoutTests/http/tests/contentextensions/block-cookies-send.https.html.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-send.html.json.
* LayoutTests/http/tests/cookies/resources/cookie-utility.py:
(delete_cookie):
* LayoutTests/http/tests/cookies/resources/resetCookies.js:
(resetCookies):
* LayoutTests/http/tests/cookies/resources/third-party-cookie-relaxing-iframe.html:
* LayoutTests/http/tests/cookies/third-party-cookie-relaxing-expected.txt:
* LayoutTests/http/tests/cookies/third-party-cookie-relaxing.html:
* LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-granted-storage-access-api.https-expected.txt: Renamed from LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-granted-storage-access-api-expected.txt.
* LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-granted-storage-access-api.https.html: Renamed from LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-granted-storage-access-api.html.
* LayoutTests/http/tests/storageAccess/aggregate-sorted-data-with-storage-access.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/aggregate-sorted-data-with-storage-access-expected.txt.
* LayoutTests/http/tests/storageAccess/aggregate-sorted-data-with-storage-access.https.html: Renamed from LayoutTests/http/tests/storageAccess/aggregate-sorted-data-with-storage-access.html.
* LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-ephemeral.html.
* LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-expected.txt.
* LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking.https.html: Renamed from LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking.html.
* LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-ephemeral.html.
* LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-expected.txt.
* LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture.https.html: Renamed from LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture.html.
* LayoutTests/http/tests/storageAccess/grant-with-prompt-under-general-third-party-cookie-blocking.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/grant-with-prompt-under-general-third-party-cookie-blocking-expected.txt.
* LayoutTests/http/tests/storageAccess/grant-with-prompt-under-general-third-party-cookie-blocking.https.html: Renamed from LayoutTests/http/tests/storageAccess/grant-with-prompt-under-general-third-party-cookie-blocking.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-ephemeral.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-from-prevalent-domain-with-user-interaction.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-from-prevalent-domain-with-user-interaction-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-from-prevalent-domain-with-user-interaction.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-from-prevalent-domain-with-user-interaction.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-true-if-third-party-has-cookies.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-true-if-third-party-has-cookies-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-true-if-third-party-has-cookies.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-true-if-third-party-has-cookies.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-ephemeral.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-ephemeral.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-ephemeral.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-but-access-from-wrong-frame.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-but-access-from-wrong-frame-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-but-access-from-wrong-frame.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-but-access-from-wrong-frame.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-without-user-interaction.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-without-user-interaction-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-without-user-interaction.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-without-user-interaction.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-nested-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-nested-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-nested-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-nested-iframe.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-detach-should-not-have-access.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-detach-should-not-have-access-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-detach-should-not-have-access.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-detach-should-not-have-access.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-cross-site-should-not-have-access.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-cross-site-should-not-have-access-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-cross-site-should-not-have-access.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-cross-site-should-not-have-access.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-same-site-should-have-access.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-same-site-should-have-access-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-same-site-should-have-access.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-same-site-should-have-access.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-with-per-page-scope-access-from-another-frame.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-with-per-page-scope-access-from-another-frame-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-with-per-page-scope-access-from-another-frame.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-with-per-page-scope-access-from-another-frame.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-with-unique-origin.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-with-unique-origin-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-with-unique-origin.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-with-unique-origin.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-without-allow-token.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-without-allow-token-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-without-allow-token.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-without-allow-token.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-iframe.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-without-allow-token.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-without-allow-token-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-without-allow-token.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-without-allow-token.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe.html.
* LayoutTests/http/tests/storageAccess/resources/get-cookies.py:
* LayoutTests/http/tests/storageAccess/resources/has-storage-access-iframe.html:
* LayoutTests/http/tests/storageAccess/resources/nesting-iframe.html:
* LayoutTests/http/tests/storageAccess/resources/produce-user-gesture-set-cookie-and-report-back.html:
* LayoutTests/http/tests/storageAccess/resources/request-storage-access-iframe.html:
* LayoutTests/http/tests/storageAccess/resources/self-navigating-frame-after-granted-access.html:
* LayoutTests/http/tests/storageAccess/resources/set-cookie.py:
* LayoutTests/http/tests/xmlhttprequest/access-control-preflight-credential-async-expected.txt:
* LayoutTests/http/tests/xmlhttprequest/access-control-preflight-credential-async.html:
* LayoutTests/http/tests/xmlhttprequest/access-control-preflight-credential-sync-expected.txt:
* LayoutTests/http/tests/xmlhttprequest/access-control-preflight-credential-sync.html:
* LayoutTests/http/tests/xmlhttprequest/cross-origin-authorization.html:
* LayoutTests/http/tests/xmlhttprequest/cross-origin-cookie-storage.html:
* LayoutTests/http/tests/xmlhttprequest/resources/cross-origin-authorization.py:
* LayoutTests/http/tests/xmlhttprequest/resources/cross-origin-no-authorization.py:
* LayoutTests/http/tests/xmlhttprequest/resources/cross-origin-set-cookies.py:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/280986@main">https://commits.webkit.org/280986@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/336771f20c31913fe8eb1cbb314068e185003d4f

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/58235 "18 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/37563 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/10718 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/61860 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/8680 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/45199 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/8874 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/47191 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/6204 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/60266 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/35215 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/50350 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/28025 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/31978 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/7647 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/7684 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/53932 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/7916 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/63565 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/2149 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/7977 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/54509 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/2157 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/50362 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/54568 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/1840 "Passed tests") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/8687 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/33392 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/34478 "Built successfully") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/35562 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/34223 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

szewai commented on LayoutTests/http/tests/cookies/third-party-cookie-relaxing.https.html:
```diff
@@ -1,6 +1,6 @@
 <html>
 <head>
-<script src="resources/resetCookies.js"></script>
+<script src="resources/resetCookiesSecure.js"></script>
```
I am not sure about the other tests but I don't think this test needs https.html -- we need https.html when the test itself needs to be hosted on security origin, not subresources. 
I know this test as I have an ongoing PR https://github.com/WebKit/WebKit/pull/28581 for it. Maybe you could merge that change in this patch, and see whether the related tests need the renaming.

charliewolfe replied:
The tests that only set cookies from an iframe don't need the mainframe to also be under https, I can change those. The tests that set cookies from the main resource do need the renaming.

I'll undo the renaming for:
third-party-cookie-relaxing.html
access-control-preflight-credential-async.html
access-control-preflight-credential-sync.html

szewai reviewed (approved):
r=me if bots are happy

charliewolfe committed:
Update more tests to conform to new SameSite cookie behavior
rdar://125572233
https://bugs.webkit.org/show_bug.cgi?id=276459

Reviewed by Matthew Finkel and Sihui Liu.

This addresses most tests that are failing due to the SameSite lax by default change in the macOS Sequoia
and iOS 18 betas. Some tests still fail and will need to be rebased when the bots are updated. Others may
require test runner changes or further test redesigns.

* LayoutTests/http/tests/contentextensions/block-cookies-send.https-expected.txt: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-send-expected.txt.
* LayoutTests/http/tests/contentextensions/block-cookies-send.https.html: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-send.html.
* LayoutTests/http/tests/contentextensions/block-cookies-send.https.html.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-send.html.json.
* LayoutTests/http/tests/cookies/resources/cookie-utility.py:
(delete_cookie):
* LayoutTests/http/tests/cookies/resources/resetCookies.js:
(resetCookies):
* LayoutTests/http/tests/cookies/resources/third-party-cookie-relaxing-iframe.html:
* LayoutTests/http/tests/cookies/third-party-cookie-relaxing-expected.txt:
* LayoutTests/http/tests/cookies/third-party-cookie-relaxing.html:
* LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-granted-storage-access-api.https-expected.txt: Renamed from LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-granted-storage-access-api-expected.txt.
* LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-granted-storage-access-api.https.html: Renamed from LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-granted-storage-access-api.html.
* LayoutTests/http/tests/storageAccess/aggregate-sorted-data-with-storage-access.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/aggregate-sorted-data-with-storage-access-expected.txt.
* LayoutTests/http/tests/storageAccess/aggregate-sorted-data-with-storage-access.https.html: Renamed from LayoutTests/http/tests/storageAccess/aggregate-sorted-data-with-storage-access.html.
* LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-ephemeral.html.
* LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking-expected.txt.
* LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking.https.html: Renamed from LayoutTests/http/tests/storageAccess/deny-due-to-no-interaction-under-general-third-party-cookie-blocking.html.
* LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-ephemeral.html.
* LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture-expected.txt.
* LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture.https.html: Renamed from LayoutTests/http/tests/storageAccess/grant-storage-access-under-opener-at-popup-user-gesture.html.
* LayoutTests/http/tests/storageAccess/grant-with-prompt-under-general-third-party-cookie-blocking.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/grant-with-prompt-under-general-third-party-cookie-blocking-expected.txt.
* LayoutTests/http/tests/storageAccess/grant-with-prompt-under-general-third-party-cookie-blocking.https.html: Renamed from LayoutTests/http/tests/storageAccess/grant-with-prompt-under-general-third-party-cookie-blocking.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-ephemeral.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-false-by-default.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-from-prevalent-domain-with-user-interaction.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-from-prevalent-domain-with-user-interaction-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-from-prevalent-domain-with-user-interaction.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-from-prevalent-domain-with-user-interaction.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-true-if-third-party-has-cookies.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-true-if-third-party-has-cookies-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-true-if-third-party-has-cookies.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-true-if-third-party-has-cookies.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-ephemeral.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-with-cookie.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-ephemeral.html.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie-expected.txt.
* LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie.https.html: Renamed from LayoutTests/http/tests/storageAccess/has-storage-access-under-general-third-party-cookie-blocking-without-cookie.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-ephemeral.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-ephemeral-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-ephemeral.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-ephemeral.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-non-sandboxed-iframe.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-but-access-from-wrong-frame.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-but-access-from-wrong-frame-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-but-access-from-wrong-frame.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-but-access-from-wrong-frame.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-without-user-interaction.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-without-user-interaction-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-without-user-interaction.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-without-user-interaction.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-nested-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-nested-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-nested-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-nested-iframe.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-detach-should-not-have-access.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-detach-should-not-have-access-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-detach-should-not-have-access.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-detach-should-not-have-access.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-cross-site-should-not-have-access.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-cross-site-should-not-have-access-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-cross-site-should-not-have-access.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-cross-site-should-not-have-access.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-same-site-should-have-access.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-same-site-should-have-access-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-same-site-should-have-access.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-then-navigate-same-site-should-have-access.html.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-with-per-page-scope-access-from-another-frame.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-with-per-page-scope-access-from-another-frame-expected.txt.
* LayoutTests/http/tests/storageAccess/request-and-grant-access-with-per-page-scope-access-from-another-frame.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-and-grant-access-with-per-page-scope-access-from-another-frame.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-with-unique-origin.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-with-unique-origin-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-with-unique-origin.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-with-unique-origin.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-without-allow-token.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-without-allow-token-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-without-allow-token.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-cross-origin-sandboxed-iframe-without-allow-token.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-iframe.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-without-allow-token.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-without-allow-token-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-without-allow-token.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-without-allow-token.html.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe.https-expected.txt: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe-expected.txt.
* LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe.https.html: Renamed from LayoutTests/http/tests/storageAccess/request-storage-access-same-origin-sandboxed-iframe.html.
* LayoutTests/http/tests/storageAccess/resources/get-cookies.py:
* LayoutTests/http/tests/storageAccess/resources/has-storage-access-iframe.html:
* LayoutTests/http/tests/storageAccess/resources/nesting-iframe.html:
* LayoutTests/http/tests/storageAccess/resources/produce-user-gesture-set-cookie-and-report-back.html:
* LayoutTests/http/tests/storageAccess/resources/request-storage-access-iframe.html:
* LayoutTests/http/tests/storageAccess/resources/self-navigating-frame-after-granted-access.html:
* LayoutTests/http/tests/storageAccess/resources/set-cookie.py:
* LayoutTests/http/tests/xmlhttprequest/access-control-preflight-credential-async-expected.txt:
* LayoutTests/http/tests/xmlhttprequest/access-control-preflight-credential-async.html:
* LayoutTests/http/tests/xmlhttprequest/access-control-preflight-credential-sync-expected.txt:
* LayoutTests/http/tests/xmlhttprequest/access-control-preflight-credential-sync.html:
* LayoutTests/http/tests/xmlhttprequest/cross-origin-authorization.html:
* LayoutTests/http/tests/xmlhttprequest/cross-origin-cookie-storage.html:
* LayoutTests/http/tests/xmlhttprequest/resources/cross-origin-authorization.py:
* LayoutTests/http/tests/xmlhttprequest/resources/cross-origin-no-authorization.py:
* LayoutTests/http/tests/xmlhttprequest/resources/cross-origin-set-cookies.py:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: https://commits.webkit.org/280986@main
